pypeit/flatfield.py

```diff
@@ -687,8 +687,16 @@ def fit(self, spat_illum_only=False, debug=False):
         # Model each slit independently
         for slit_idx, slit_spat in enumerate(self.slits.spat_id):
             # Is this a good slit??
-            if self.slits.mask[slit_idx] != 0:
+            if self.flatpar['pixelflat_min_wave'] is not None or self.flatpar['pixelflat_max_wave'] is not None:
```

Comment:
This block could be shortened to read: 

good_bpms = [0,2] if ( )  else [0, 2, 8, 10]